com.zsmartsystems.zigbee.dongle.ember/src/main/java/com/zsmartsystems/zigbee/dongle/ember/internal/ash/AshFrameHandler.java

```diff
@@ -202,8 +202,10 @@ public void run() {
                                         if (response == null) {
                                             logger.debug("ASH: No frame handler created for {}", packet);
                                         } else {
-                                            notifyTransactionComplete(response);
-                                            handleIncomingFrame(response);
+                                            if (!notifyTransactionComplete(response)) {
```

Comment:
Before, the result notifyTransactionComplete was ignored, and the response message unconditionally on to handleIncomingFrame, even if it was a (completing) match.